### ISSUE TITLE
Chat Feature between the Players

### ISSUE THREAD

**Issue by Airbornharsh** [code100x/chess#226]

**Comment by Airbornharsh** [comment]
@hkirat Check this out

**Comment by janareddyc7** [comment]
i dont think this works beacuse we can create a system where you chat with the friend id or friendship system like chess.com becuase we cant save that data in the db

**Comment by Airbornharsh** [comment]
I have not stored the chats anywhere for now.
